src/ml_tooling/plots/target_feature_distribution.py

```diff
@@ -0,0 +1,72 @@
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+import pandas as pd
+import numpy as np
+
+from ml_tooling.utils import DataType
+from scipy import stats
+
+
+def target_feature_distribution(
+    data: pd.DataFrame,
+    targets: DataType,
+    features: DataType,
+    title: str = None,
+    labels: list = None,
+    median: bool = False,
```

Comment:
The signature should match the other plot functions to the extent it makes sense - e.g top_n, bottom_n might not make sense, but we definitely want an ax argument.